### PR TITLE
Feat/new faq

### PR DESCRIPTION
**Pull request by lorycade**
add new faqs

### CODE REVIEW

**Review by emnlpstr — APPROVED** [review]

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -109,59 +112,112 @@ questions:
       - title: Come si revoca un'utenza?
         content: Il rappresentante legale o l'incaricato, dopo aver selezionato la PA per la quale intende operare, accede, attraverso il menù a tendina che si apre facendo click sul suo nome, alla sezione "Gli utenti della tua PA". In questa sezione visualizza la lista degli utenti collegati alla propria amministrazione e facendo click sull'icona "Revoca" può revocare l'utenza.
       - title: Sono previste funzionalità per supportare la partecipazione di forme aggregative come, ad esempio, le unioni di comuni?
-        content: Si. All’interno della piattaforma l’utente con il ruolo di “incaricato” ha facoltà di operare per più amministrazioni. Il ruolo di "incaricato" può essere assegnato a un utente solo dal rappresentante legale dell'amministrazione. 
+        content: Si. All’interno della piattaforma l’utente con il ruolo di “incaricato” ha facoltà di operare per più amministrazioni. Il ruolo di "incaricato" può essere assegnato a un utente solo dal rappresentante legale dell'amministrazione.
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:115]:
```suggestion
        content: Si. All'interno della piattaforma l’utente con il ruolo di “incaricato” ha facoltà di operare per più amministrazioni. Il ruolo di "incaricato" può essere assegnato a un utente solo dal rappresentante legale dell'amministrazione.
```

File: contents/faq-page/faq.yml [hunk 2]
```diff
@@ -109,59 +112,112 @@ questions:
...
+      - title: Di quante modalità di migrazione può avvalersi un Comune al fine di rispondere correttamente ai requisiti previsti dall'Avviso riferito alla Misura 1.2?
+        content: Il Comune potrà effettuare la migrazione avvalendosi dei due modelli di migrazione ""Trasferimento in sicurezza dell’infrastruttura IT"" e ""Aggiornamento in sicurezza di applicazioni in Cloud"" come delineato nella Strategia Nazionale per il Cloud.Per ulteriori approfondimenti si rimanda all'Allegato 2 corrispondente alla fascia di popolazione del tuo Comune.
+      - title: Quali sono le caratteristiche del modello di migrazione "Trasferimento in sicurezza dell’infrastruttura IT"?
+        content: L’opzione "Trasferimento in sicurezza dell’infrastruttura IT" consente di sfruttare la strategia di migrazione Lift&Shift (anche detta Rehost), cioè la migrazione al Cloud dell’infrastruttura già esistente, senza la necessità di reingegnerizzare le applicazioni. Tale modalità consiste nel migrare l’intero servizio, comprensivo di applicazioni e dati su un hosting cloud senza apportare modifiche agli applicativi, ovvero replicando il servizio esistente in un ambiente cloud.
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:179]:
```suggestion
        content: L’opzione "Trasferimento in sicurezza dell'infrastruttura IT" consente di sfruttare la strategia di migrazione Lift&Shift (anche detta Rehost), cioè la migrazione al Cloud dell’infrastruttura già esistente, senza la necessità di reingegnerizzare le applicazioni. Tale modalità consiste nel migrare l’intero servizio, comprensivo di applicazioni e dati su un hosting cloud senza apportare modifiche agli applicativi, ovvero replicando il servizio esistente in un ambiente cloud.
```

File: contents/faq-page/faq.yml [hunk 3]
```diff
@@ -109,59 +112,112 @@ questions:
...
+      - title: Un Comune è obbligato ad adottare la stessa modalità di migrazione per tutti i servizi?
+        content: No, il Comune può decidere singolarmente per ognuno dei servizi oggetto della migrazione. L’Ente potrà selezionare il modello di migrazione più adatto da presentare all’interno del piano di migrazione.
+      - title: Se ho già migrato un servizio in cloud in modalità "Trasferimento in sicurezza dell’infrastruttura IT" posso aderire al finanziamento per implementare la modalità "Aggiornamento in sicurezza di applicazioni in Cloud"?
+        content: Si, se il Comune ha già migrato un servizio in cloud in modalità "Trasferimento in sicurezza dell’infrastruttura IT" può accedere al finanziamento previsto dall'Avviso, riferito alla Misura 1.2, per l'implementazione della modalità "Aggiornamento in sicurezza di applicazioni in Cloud".In ogni caso, il processo di migrazione al cloud deve essere stato realizzato a decorrere dal 01 Febbraio 2020 con risorse finanziarie proprie.
+      - title: Il Canone del primo anno è incluso nel finanziamento? 
+        content: Si, il canone del primo anno è incluso nell’importo forfettario come definito dagli allegati 2.
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:185]:
```suggestion
        content: Si, il canone del primo anno è incluso nell'importo forfettario come definito dagli allegati 2.
```

File: contents/faq-page/faq.yml [hunk 4]
```diff
@@ -109,59 +112,112 @@ questions:
...
+        linkLabel: Ulteriori informazioni sono disponibili al seguente link
+        link: 'https://innovazione.gov.it/dipartimento/focus/strategia-cloud-italia/'
+        ariaLabel: Vedi ulteriori informazioni sulla strategia cloud italia (Collegamento esterno - Apre su nuova scheda)
+      - title: Quali sono gli obiettivi relativi a Cloud e Infrastrutture digitali nell'ambito del Piano Nazionale di Ripresa e Resilienza?
+        content: La trasformazione digitale della Pubblica Amministrazione (PA) segue un approccio “cloud first”, orientato alla migrazione dei dati e degli applicativi informatici delle singole amministrazioni verso un ambiente cloud.Il processo consentirà di razionalizzare e consolidare molti dei data center oggi distribuiti sul territorio, a partire da quelli meno efficienti e sicuri. Ad oggi il 95% dei circa 11mila data center utilizzati dagli enti pubblici italiani presenta carenze nei requisiti minimi di sicurezza, affidabilità, capacità elaborativa ed efficienza.
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:194]:
```suggestion
        content: La trasformazione digitale della Pubblica Amministrazione (PA) segue un approccio “cloud first”, orientato alla migrazione dei dati e degli applicativi informatici delle singole amministrazioni verso un ambiente cloud. Il processo consentirà di razionalizzare e consolidare molti dei data center oggi distribuiti sul territorio, a partire da quelli meno efficienti e sicuri. Ad oggi il 95% dei circa 11 mila data center utilizzati dagli enti pubblici italiani presenta carenze nei requisiti minimi di sicurezza, affidabilità, capacità elaborativa ed efficienza.
```

File: contents/faq-page/faq.yml [hunk 5]
```diff
@@ -109,59 +112,112 @@ questions:
...
-        content: "L’importo del finanziamento concedibile ai Comuni è individuato in un importo forfettario determinato secondo il seguente criterio:<br></br>a. per i Comuni fino a 5.000 ab.: pagoPA: €607 per servizio (min.3 max 111);<br>b. per i Comuni 5.001 - 20.000 ab. pagoPA: €857 per servizio (min.3 max 111);<br>c. per i Comuni 20.001 - 100.000 ab. pagoPA: €1821 per servizio (min.5 max 111);<br>d. per i Comuni 100.001 - 250.000 ab. pagoPA: €2747 per servizio (min.5 max 111);<br>e. per i Comuni > 250.000 ab. pagoPA: €7967 per servizio (min.5 max 111)."
+        content: 'L’importo del finanziamento concedibile ai Comuni è individuato in un importo forfettario determinato secondo il seguente criterio:<br></br>a. per i Comuni fino a 5.000 ab.: pagoPA: €607 per servizio (min.3 max 111);<br>b. per i Comuni 5.001 - 20.000 ab. pagoPA: €857 per servizio (min.3 max 111);<br>c. per i Comuni 20.001 - 100.000 ab. pagoPA: €1821 per servizio (min.5 max 111);<br>d. per i Comuni 100.001 - 250.000 ab. pagoPA: €2747 per servizio (min.5 max 111);<br>e. per i Comuni > 250.000 ab. pagoPA: €7967 per servizio (min.5 max 111).'
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:220]:
```suggestion
        content: "L'importo del finanziamento concedibile ai Comuni è individuato in un importo forfettario determinato secondo il seguente criterio:<br></br>a. per i Comuni fino a 5.000 ab.: pagoPA: €607 per servizio (min.3 max 111);<br>b. per i Comuni 5.001 - 20.000 ab. pagoPA: €857 per servizio (min.3 max 111);<br>c. per i Comuni 20.001 - 100.000 ab. pagoPA: €1821 per servizio (min.5 max 111);<br>d. per i Comuni 100.001 - 250.000 ab. pagoPA: €2747 per servizio (min.5 max 111);<br>e. per i Comuni > 250.000 ab. pagoPA: €7967 per servizio (min.5 max 111)."
```

File: contents/faq-page/faq.yml [hunk 6]
```diff
@@ -191,9 +247,9 @@ questions:
...
-        content: "L’importo del finanziamento concedibile ai Comuni è individuato in un importo forfettario determinato secondo il seguente criterio:<br></br>a. per i Comuni fino a 5.000 ab.: app IO: €243 per servizio (min.3 max 50);<br>b. per i Comuni 5.001 - 20.000 ab. app IO: €343 per servizio (min.3 max 50);<br>c. per i Comuni 20.001 - 100.000 ab. app IO: €728 per servizio (min.5 max 50);<br>d. per i Comuni 100.001 - 250.000 ab. app IO: €1.099 per servizio (min.5 max 50);<br>e. per i Comuni > 250.000 ab. app IO: €3.187 per servizio (min.5 max 50)."
+        content: 'L’importo del finanziamento concedibile ai Comuni è individuato in un importo forfettario determinato secondo il seguente criterio:<br></br>a. per i Comuni fino a 5.000 ab.: app IO: €243 per servizio (min.3 max 50);<br>b. per i Comuni 5.001 - 20.000 ab. app IO: €343 per servizio (min.3 max 50);<br>c. per i Comuni 20.001 - 100.000 ab. app IO: €728 per servizio (min.5 max 50);<br>d. per i Comuni 100.001 - 250.000 ab. app IO: €1.099 per servizio (min.5 max 50);<br>e. per i Comuni > 250.000 ab. app IO: €3.187 per servizio (min.5 max 50).'
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:250]:
```suggestion
        content: "L'importo del finanziamento concedibile ai Comuni è individuato in un importo forfettario determinato secondo il seguente criterio:<br></br>a. per i Comuni fino a 5.000 ab.: app IO: €243 per servizio (min.3 max 50);<br>b. per i Comuni 5.001 - 20.000 ab. app IO: €343 per servizio (min.3 max 50);<br>c. per i Comuni 20.001 - 100.000 ab. app IO: €728 per servizio (min.5 max 50);<br>d. per i Comuni 100.001 - 250.000 ab. app IO: €1.099 per servizio (min.5 max 50);<br>e. per i Comuni > 250.000 ab. app IO: €3.187 per servizio (min.5 max 50)."
```

File: contents/faq-page/faq.yml [hunk 7]
```diff
@@ -215,12 +271,12 @@ questions:
       - title: Qual è l'importo del finanziamento concedibile?
         content: L'importo del finanziamento riconoscibile è pari a €14.000 al termine della realizzazione del progetto.
       - title: Sono previste risorse espressamente dedicate ai territori delle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia?
-        content: Si, l'Avviso destina il 40% delle risorse al finanziamento di Soggetti Attuatori ubicati nelle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia, nel rispetto della priorità trasversale del PNRR relativa al “superamento dei divari territoriali” di cui all’art. 2 comma 6 bis del decreto- legge 31 maggio 2021 n.77, convertito in legge 29 luglio 2021, n. 108. 
+        content: Si, l'Avviso destina il 40% delle risorse al finanziamento di Soggetti Attuatori ubicati nelle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia, nel rispetto della priorità trasversale del PNRR relativa al “superamento dei divari territoriali” di cui all’art. 2 comma 6 bis del decreto- legge 31 maggio 2021 n.77, convertito in legge 29 luglio 2021, n. 108.
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:274]:
```suggestion
        content: Sì, l'Avviso destina il 40% delle risorse al finanziamento di Soggetti Attuatori ubicati nelle regioni Abruzzo, Basilicata, Campania, Calabria, Molise, Puglia, Sardegna, Sicilia, nel rispetto della priorità trasversale del PNRR relativa al “superamento dei divari territoriali” di cui all’art. 2 comma 6 bis del decreto- legge 31 maggio 2021 n.77, convertito in legge 29 luglio 2021, n. 108.
```

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: contents/faq-page/faq.yml (hunk 1 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml (hunk 2 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml (hunk 3 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml (hunk 4 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml (hunk 5 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml (hunk 6 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml (hunk 7 above)
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml [hunk 8]
```diff
@@ -249,36 +305,38 @@ questions:
...
-      - title: "Ho ricevuto una notifica e una PEC con richieste di integrazione alla classificazione dati e servizi da parte dell’Agenzia per la Cybersicurezza Nazionale (ACN): cosa devo fare?"
+        content: 'L’Agenzia per la Cybersicurezza Nazionale fornisce un riscontro sulla conformità dell’elenco dei servizi e della classificazione entro 90 giorni dalla ricezione della documentazione. L’Agenzia può prorogare questo termine nel caso siano necessari degli approfondimenti: il termine viene posticipato al massimo per ulteriori 30 giorni, e in ogni caso non più di una volta. Se invece è necessario richiedere alla PA integrazioni o informazioni aggiuntive, la procedura viene interrotta e i termini cominciano a decorrere nuovamente dal momento in cui vengono ricevute le informazioni richieste, che devono essere fornite entro 30 giorni. Nel caso non dovesse esserci nessun riscontro entro 90 giorni o, nel caso di posticipo, entro 120 giorni, l’elenco e la classificazione dei dati e dei servizi sono convalidati.'
```
[the remark this answers is not in the document]
Comment (emnlpstr) [reply contents/faq-page/faq.yml]: fixed

File: contents/faq-page/faq.yml [hunk 9]
```diff
@@ -70,7 +73,7 @@ questions:
...
       - title: È possibile accedere alla piattaforma anche quando non si ricopre il ruolo di rappresentante legale dell'amministrazione?
-        content: Si. Nel caso in cui a effettuare l'accesso non sia direttamente il rappresentante legale ma una persona da lui incaricata, è presente una casella da spuntare per dichiarare di stare agendo per conto del rappresentante legale dell'amministrazione. Alla fine della procedura di primo accesso, il rappresentante legale della PA riceve una PEC con cui viene a informato del fatto che la persona incaricata ha agito per suo conto su PA digitale 2026. 
+        content: Si. Nel caso in cui a effettuare l'accesso non sia direttamente il rappresentante legale ma una persona da lui incaricata, è presente una casella da spuntare per dichiarare di stare agendo per conto del rappresentante legale dell'amministrazione. Alla fine della procedura di primo accesso, il rappresentante legale della PA riceve una PEC con cui viene a informato del fatto che la persona incaricata ha agito per suo conto su PA digitale 2026.
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:76]:
```suggestion
        content: Sì. Nel caso in cui a effettuare l'accesso non sia direttamente il rappresentante legale ma una persona da lui incaricata, è presente una casella da spuntare per dichiarare di stare agendo per conto del rappresentante legale dell'amministrazione. Alla fine della procedura di primo accesso, il rappresentante legale della PA riceve una PEC con cui viene a informato del fatto che la persona incaricata ha agito per suo conto su PA digitale 2026.
```

**Pull request merged**